dashboard/test/unit/lib/models/run.test.js

```diff
@@ -0,0 +1,130 @@
+const expect = require('chai').expect;
+
+// unit under test
+const Run = require('../../../../lib/models/run');
+
+describe('Run class', () => {
+    let currTime = Date.now();
+
+    const mockRunData = {
+        command: 'abc',
+        id: 'id',
+        startTime: currTime,
+    };
+    let run;
+
+    before(() => {
+        run = new Run(mockRunData);
+    });
+
+    it('should create the instance correctly', () => {
+        expect(run.command).to.equal('abc');
+        expect(run.id).to.equal('id');
+        expect(run.startTime).to.equal(currTime);
+
+        expect(run.endTime).to.equal(0);
+        expect(run.events.length).to.equal(0);
+        expect(run.cpuUsage.length).to.equal(0);
+        expect(run.memoryUsage.length).to.equal(0);
+    });
+
+    describe('Setters and getters', () => {
```

Comment:
Don't need to add a separate `describe` block for "setters and getters". Could just be:
```javascript
describe('~setPaused', () => {
```